middleware/src/main/java/com/frontegg/sdk/middleware/permission/IPermissionEvaluator.java

```diff
@@ -0,0 +1,10 @@
+package com.frontegg.sdk.middleware.permission;
+
+import com.frontegg.sdk.middleware.context.FronteggContext;
+
+import javax.servlet.http.HttpServletRequest;
+
+public interface IPermissionEvaluator {
```

Comment:
1. Please dont use I prefix
2. I think we agreed to remove permission handling, right @aviadmizrachi